cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/contactpoint/HttpClusterBootstrapRoutes.scala

```diff
@@ -39,17 +39,10 @@ final class HttpClusterBootstrapRoutes(settings: ClusterBootstrapSettings) exten
     }
   }
 
-  // TODO ip whitelist feature?
-  val routes = {
-    // TODO basePath, same as akka-management
-    // val basePath = if (pathPrefixName.isEmpty) rawPathPrefix(pathPrefixName) else pathPrefix(pathPrefixName)
```

Comment:
the base path things are now only in the akka-management, since it's "basepath" for all the APIs, we can add more if needed though, in each of the route providers